core/domain/exp_domain.py

```diff
@@ -3157,6 +3157,70 @@ def _convert_states_v18_dict_to_v19_dict(cls, states_dict):
 
         return states_dict
 
+    @classmethod
+    def _convert_states_v19_dict_to_v20_dict(cls, states_dict):
+        """Converts from version 19 to 20. Version 20 converts non textangular
+        html content to the textangular html content.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+        for state_dict in states_dict.values():
+
+            state_dict['content']['html'] = utils.convert_to_text_angular(
+                state_dict['content']['html'])
+            if state_dict['interaction']['default_outcome']:
+                state_dict[
```

Comment:
For readability, it would be better to assign `state_dict['interaction']['default_outcome']['feedback']['html']` to a variable (say `interaction_feedback_html` and then use that rather than splitting across so many lines as you do here.

Reply:
Done!